drivers/clock_control/clock_control_mcux_ccm.c

```diff
@@ -38,55 +36,59 @@ static int mcux_ccm_get_subsys_rate(struct device *dev,
 				    clock_control_subsys_t sub_system,
 				    u32_t *rate)
 {
-	u32_t clock_name = (u32_t) sub_system;
+	u32_t clock_name = (u32_t)sub_system;
 
 	switch (clock_name) {
-
 #ifdef CONFIG_I2C_MCUX_LPI2C
 	case IMX_CCM_LPI2C_CLK:
 		if (CLOCK_GetMux(kCLOCK_Lpi2cMux) == 0) {
-			*rate = CLOCK_GetPllFreq(kCLOCK_PllUsb1) / 8
-				/ (CLOCK_GetDiv(kCLOCK_Lpi2cDiv) + 1);
+			*rate = CLOCK_GetPllFreq(kCLOCK_PllUsb1) / 8 /
+				(CLOCK_GetDiv(kCLOCK_Lpi2cDiv) + 1);
 		} else {
-			*rate = CLOCK_GetOscFreq()
-				/ (CLOCK_GetDiv(kCLOCK_Lpi2cDiv) + 1);
+			*rate = CLOCK_GetOscFreq() /
+				(CLOCK_GetDiv(kCLOCK_Lpi2cDiv) + 1);
 		}
 
 		break;
 #endif
 
-	case IMX_CCM_LPSPI_CLK:
-	{
+	case IMX_CCM_LPSPI_CLK: {
 		u32_t lpspi_mux = CLOCK_GetMux(kCLOCK_LpspiMux);
 		clock_name_t lpspi_clock = lpspi_clocks[lpspi_mux];
 
-		*rate = CLOCK_GetFreq(lpspi_clock)
-			/ (CLOCK_GetDiv(kCLOCK_LpspiDiv) + 1);
+		*rate = CLOCK_GetFreq(lpspi_clock) /
+			(CLOCK_GetDiv(kCLOCK_LpspiDiv) + 1);
 		break;
 	}
 
 	case IMX_CCM_LPUART_CLK:
 		if (CLOCK_GetMux(kCLOCK_UartMux) == 0) {
-			*rate = CLOCK_GetPllFreq(kCLOCK_PllUsb1) / 6
-				/ (CLOCK_GetDiv(kCLOCK_UartDiv) + 1);
+			*rate = CLOCK_GetPllFreq(kCLOCK_PllUsb1) / 6 /
+				(CLOCK_GetDiv(kCLOCK_UartDiv) + 1);
 		} else {
-			*rate = CLOCK_GetOscFreq()
-				/ (CLOCK_GetDiv(kCLOCK_UartDiv) + 1);
+			*rate = CLOCK_GetOscFreq() /
+				(CLOCK_GetDiv(kCLOCK_UartDiv) + 1);
 		}
 
 		break;
 
 #ifdef CONFIG_DISK_ACCESS_USDHC1
 	case IMX_CCM_USDHC1_CLK:
 		*rate = CLOCK_GetSysPfdFreq(kCLOCK_Pfd0) /
-				(CLOCK_GetDiv(kCLOCK_Usdhc1Div) + 1U);
+			(CLOCK_GetDiv(kCLOCK_Usdhc1Div) + 1U);
 		break;
 #endif
 
 #ifdef CONFIG_DISK_ACCESS_USDHC2
 	case IMX_CCM_USDHC2_CLK:
 		*rate = CLOCK_GetSysPfdFreq(kCLOCK_Pfd0) /
-				(CLOCK_GetDiv(kCLOCK_Usdhc2Div) + 1U);
+			(CLOCK_GetDiv(kCLOCK_Usdhc2Div) + 1U);
+		break;
+#endif
+
+#ifdef CONFIG_DMA_IMX_RT
+	case IMX_CCM_EDMA_CLK:
+		*rate = CLOCK_GetIpgFreq();
 		break;
 #endif
```

Comment:
Please make the clock control changes in two separate commits - 1 commit for the whitespace changes, 1 commit to add the EDMA clock.

Reply:
Ok

Reply:
These changes still appear in the same commit as the dma driver. Please move them to new commits; you can keep them in the same pull request.

Reply:
done